saleor_app_framework/app_framework/core/install.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import secrets
+import string
+from typing import Awaitable, Callable, List
+
+from ..core.conf import settings
+from .errors import InstallAppError
+from .graphql import GraphqlError, get_executor, get_saleor_api_url
+from .types import AppToken, DomainName, Url, WebhookData
+
+logger = logging.getLogger(__file__)
+
+
+CREATE_WEBHOOK = """
+mutation WebhookCreate($input: WebhookCreateInput!){
+    webhookCreate(input: $input){
+        webhookErrors{
+         field
+         message
+         code
+       }
+       webhook {
+             id
+       }
+   }
+}
```

Comment:
Maybe we could use query formated in playground? 
```suggestion
mutation WebhookCreate($input: WebhookCreateInput!) {
  webhookCreate(input: $input) {
    webhookErrors {
      field
      message
      code
    }
    webhook {
      id
    }
  }
}

```